src/injectscripts/jobs/situational_awareness.js

```diff
@@ -143,6 +156,18 @@ if (developmentMode) {
     loadAircraftLastKnownPositions();
 }
 
+/**
+ * Sends the selected HQs & start/end date range to the content script.
+ */
+function sendStateToContentScript() {
+    let params = {
+        hqs: contentViewModel.filterViewModel.selectedEntities.peek(),
+        startDate: contentViewModel.filterViewModel.startDate().toDate(),
```

Comment:
This has issues when time range is set to 'current', it returns null. seems to be a weird bug in beacon but we will need to work with it.

On a clean page load these will be null, even if a date it selected. very stange.